<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FractionCalculator</class>
 <widget class="QMainWindow" name="FractionCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>246</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>FractionCalculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Fraction Calculator by Travis Stephney</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_3">
       <widget class="QLineEdit" name="Num1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="Num2">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>20</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="Denom1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="Denom2">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>60</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="Calculate">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>120</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Calculate</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Exit">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>140</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
       <widget class="QComboBox" name="Operation">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>40</y>
          <width>65</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Add</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Subtract</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Multiply</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Divide</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="Answer">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>90</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>423</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
